game/src/test/scala/io/aigar/model/PlayerModelSpec.scala

```diff
@@ -0,0 +1,26 @@
+package io.aigar.model
```

Comment:
Small tip, when you rename a file do : `git mv old new`. It will give you better diffs